clients/gtest/gbmv_gtest.cpp

```diff
@@ -39,7 +39,8 @@ namespace
             switch(GBMV_TYPE)
             {
             case GBMV:
-                return !strcmp(arg.function, "gbmv") || !strcmp(arg.function, "gbmv_bad_arg");
+                return !strcmp(arg.function, "gbmv") || !strcmp(arg.function, "gbmv_bad_arg")
+                       || !strcmp(arg.function, "gbmv_arg_check");
```

Comment:
Does `arg_check` need to be its own category, since it is just a combination of certain parameters and/or NaNs with a certain expected output?

Unlike `bag_arg`, I'm not sure we need to separate `arg_check` into its separate category.

Reply:
Not necessarily. An alternative would be to split up the yaml categories into a sub-group for these edge cases. This is fine with me if we prefer that.

Reply:
I prefer separating out the YAML categories.